README.md

```diff
@@ -141,16 +141,49 @@ Check out the [upgrade guide](doc/upgrade-cluster.md).
 ```
 $ helm delete pravega-bk --purge
 ```
-
-### Uninstall the Operator
-> Note that the Bookkeeper clusters managed by the Bookkeeper operator will NOT be deleted even if the operator is uninstalled.
-
-If you want to delete the Bookkeeper cluster, make sure to do it before uninstalling the operator. Also, once the Bookkeeper cluster has been deleted, make sure to check that the zookeeper metadata has been cleaned up before proceeding with the deletion of the operator. This can be confirmed with the presence of the following log message in the operator logs.
+Once the Bookkeeper cluster has been deleted, make sure to check that the zookeeper metadata has been cleaned up before proceeding with the deletion of the operator. This can be confirmed with the presence of the following log message in the operator logs.
 ```
 zookeeper metadata deleted
 ```
 
-You can then delete the operator
+In case the operator fails to delete metadata from the zookeeper pods, it will send out a `ZKMETA_CLEANUP_ERROR` event to notify the user about this failure. The user can check this event by doing `kubectl get events`. The following is the sample describe output of the event that is generated by the operator in such a case
```

Comment:
Please also specify the log message generated in BK operator logs, if metadata deletion fails.

Reply:
done